backup-codelab-done/app/build.gradle

```diff
@@ -35,7 +35,7 @@ android {
 
 dependencies {
     implementation fileTree(dir: 'libs', include: ['*.jar'])
-    implementation 'com.android.support:appcompat-v7:26.0.1'
-    implementation 'com.android.support:design:26.0.1'
-    implementation 'com.android.support:support-v4:26.0.1'
+    implementation 'com.android.support:appcompat-v7:26.1.0'
```

Comment:
A lot of the samples I've worked on make the version number a var in an ext block

Reply:
skipping since it's easier to use IDE version bump feature without var.